content/guides/caching.md

```diff
@@ -4,284 +4,358 @@ sort: 11
-      filename: "chunk-manifest.json",
-      manifestVariable: "webpackManifest"
-    })
-  ]
-};
+    return element;
+  }
+
+  document.body.appendChild(component());
 ```
 
-As we removed the manifest from the entry chunk, now it’s our responsibility to provide webpack with it. The `manifestVariable` option in the example above is the name of the global variable where webpack will look for the manifest JSON. This *should be defined before we require our bundle in HTML*. This is achieved by inlining the contents of the JSON in HTML. Our HTML head section should look like this:
-
-```html
-<html>
-  <head>
-    <script>
-    //<![CDATA[
-    window.webpackManifest = {"0":"main.5f020f80c23aa50ebedf.js","1":"vendor.81adc64d405c8b218485.js"}
-    //]]>
-    </script>
-  </head>
-  <body>
-  </body>
-</html>
+Running another build, we would expect only our `main` bundle's hash to change, however...
+
+``` bash
+Hash: d38a06644fdbb898d795
+Version: webpack 3.3.0
+Time: 1445ms
+                          Asset       Size  Chunks                    Chunk Names
+ vendor.a7561fb0e9a071baadb9.js     541 kB       0  [emitted]  [big]  vendor
+   main.b746e3eb72875af2caa9.js    1.22 kB       1  [emitted]         main
+runtime.1400d5af64fc1b7b3a45.js    5.85 kB       2  [emitted]         runtime
+                     index.html  352 bytes          [emitted]
+   [1] ./src/index.js 421 bytes {1} [built]
+   [2] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [3] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [4] ./src/print.js 62 bytes {1} [built]
+   [5] multi lodash 28 bytes {0} [built]
+    + 1 hidden module
 ```
 
-At the end of the day, the hashes for the files should be based on the file content. For this use [webpack-chunk-hash](https://github.com/alexindigo/webpack-chunk-hash) or [webpack-md5-hash](https://github.com/erm0l0v/webpack-md5-hash).
-
-So the final `webpack.config.js` should look like this:
-
-```js
-var path = require("path");
-var webpack = require("webpack");
-var ChunkManifestPlugin = require("chunk-manifest-webpack-plugin");
-var WebpackChunkHash = require("webpack-chunk-hash");
-
-module.exports = {
-  entry: {
-    vendor: "./src/vendor.js", // vendor reference file(s)
-    main: "./src/index.js" // application code
-  },
-  output: {
-    path: path.join(__dirname, "build"),
-    filename: "[name].[chunkhash].js",
-    chunkFilename: "[name].[chunkhash].js"
-  },
-  plugins: [
-    new webpack.optimize.CommonsChunkPlugin({
-      name: ["vendor", "manifest"], // vendor libs + extracted manifest
-      minChunks: Infinity,
-    }),
-    new webpack.HashedModuleIdsPlugin(),
-    new WebpackChunkHash(),
-    new ChunkManifestPlugin({
-      filename: "chunk-manifest.json",
-      manifestVariable: "webpackManifest",
-      inlineManifest: true
-    })
-  ]
-};
+... we can see that all three have. This is because the [`module.id`](/api/module-variables#module-id-commonjs-) of each is incremental by default, so...
```

Comment:
No I did mean "incremental" in the sense that modules ids are simply incremented up in the order they are resolved. However I do see how this could be a bit confusing... I'll rephrase.